kie-soup-dataset/kie-soup-dataset-sql-tests/kie-soup-dataset-sql-tests-db2/src/test/java/org/dashbuilder/dataprovider/sql/DB2TestSettings.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.dataprovider.sql;
+
+import javax.sql.DataSource;
+
+import com.ibm.db2.jcc.DB2SimpleDataSource;
+import org.apache.commons.lang3.StringUtils;
+import org.dashbuilder.dataset.def.SQLDataSetDef;
+
+public class DB2TestSettings extends DatabaseTestSettings {
+
+    @Override
+    public String getDatabaseType() {
+        return DB2;
+    }
+
+    @Override
+    public SQLDataSourceLocator getDataSourceLocator() {
+        return new SQLDataSourceLocator() {
+            public DataSource lookup(SQLDataSetDef def) throws Exception {
+                String server = connectionSettings.getProperty("server");
+                String database = connectionSettings.getProperty("database");
+                String port = connectionSettings.getProperty("port");
+                String user = connectionSettings.getProperty("user");
+                String password = connectionSettings.getProperty("password");
+
```

Comment:
Let's provide `datasource.properties` ad `filtered-resources`

`maxPoolSize=${maven.btm.maxPoolSize}
allowLocalTransactions=true
# JDBC/Database properties that are set in the maven pom
#
#   the below variable names (i.e. "${maven.datasource.classname}) are
# automagically replaced with their values (defined in the pom.xml)
# because of the fact that <filtering> is set to true in for the
# src/test/resources directory in the pom.
#
className=${maven.datasource.classname}
driverClassName=${maven.jdbc.driver.class}
user=${maven.jdbc.username}
password=${maven.jdbc.password}
url=${maven.jdbc.url}
serverName=${maven.jdbc.db.server}
portNumber=${maven.jdbc.db.port}
databaseName=${maven.jdbc.db.name}
defaultSchema=${maven.jdbc.schema}
dbBaseDir=${project.build.directory}
makeBaseDb=false
testMarshalling=false`

Reply:
by default these should use H2